server/src/main/xsd/gateway-config-201409.xsd

```diff
@@ -476,7 +476,7 @@
                      type="gateway:CollapsedString"/>
 
             <element maxOccurs="unbounded" minOccurs="0" name="accept"
-                     type="anyURI"/>
```

Comment:
The implementation should work with xsd:anyURI also; had added the new type in order to be able to define specific patterns, the implementation should work with this reverted also.